gpMgmt/bin/go-tools/cli/commands.go

```diff
@@ -0,0 +1,108 @@
+package cli
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/greenplum-db/gp-common-go-libs/gplog"
+	"github.com/greenplum-db/gpdb/gp/constants"
+	"github.com/greenplum-db/gpdb/gp/hub"
+	"github.com/greenplum-db/gpdb/gp/idl"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+)
+
+var (
+	ReadFile        = os.ReadFile
+	Unmarshal       = json.Unmarshal
+	DialContextFunc = grpc.DialContext
+	ConnectToHub    = ConnectToHubFn
+
+	ConfigFilePath string
+	Conf           *hub.Config
+
+	Verbose bool
+)
+
+func RootCommand() *cobra.Command {
+	root := &cobra.Command{
+		Use: "gp",
+	}
+
+	root.PersistentFlags().StringVar(&ConfigFilePath, "config-file", filepath.Join(os.Getenv("GPHOME"), constants.ConfigFileName), `Path to gp configuration file`)
+	root.PersistentFlags().BoolVar(&Verbose, "verbose", false, `Provide verbose output`)
+
+	root.AddCommand(agentCmd())
```

Comment:
Was this considered?

```suggestion
	root.AddCommand(
		agentCmd(),
		hubCmd(),
		configureCmd(),
		startCmd(),
		statusCmd(),
		stopCmd(),
	)
```

Reply:
Updated to this format.